src/kudu/consensus/quorum_util.cc

```diff
@@ -139,6 +139,27 @@ int MajoritySize(int num_voters) {
   return (num_voters / 2) + 1;
 }
 
+int ResolveCommitRequirement(
+    int total_voters, const std::string& commit_req) {
+  int num_votes_required =
+      ParseCommitRequirement(commit_req);
+  // Resolve majority specification.
+  if (num_votes_required != -1) {
+    DCHECK_GE(total_voters, num_votes_required);
```

Comment:
seems to me too strict a check.if for some reason prod_config passes in that
we neeed 2 votes for region but there is only 1 voter per region, did we really mean
min 2 voters, or did we mean 2 voters if 2 are available.

Reply:
I've always considered minimum 2. Interpreting it as 
> 2 voters if 2 are available

leaves a possibility of less than 2 is fine, which is NOT safe.